core/src/test/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunctionTest.java

```diff
@@ -81,6 +81,13 @@ private static Stream<Arguments> testLogDoubleArguments() {
     return builder.add(Arguments.of(2D, 2D)).build();
   }
 
+  private static Stream<Arguments> testLogInvalidDoubleArguments() {
+    Stream.Builder<Arguments> builder = Stream.builder();
+    return builder.add(Arguments.of(0D, -2D))
+        .add(Arguments.of(0D, 2D))
+        .add(Arguments.of(2D, 0D)).build();
```

Comment:
You can do
```java
return Stream.of(Arguments.of(...), Arguments.of(...), ...);
```

Reply:
Thanks! Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/225/commits/5be477ea1d48a399ee00836694f61b3444dd1dac